src/objects.js

```diff
@@ -1,29 +1,39 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {
+    name: name,
+    age: age,
+  };
+  return person;
 };
 
 const getName = (object) => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = (person) => {
-  // your code here
+  if (person.age > 65) {
+    return true;
+  }
+  return false;
 };
 
 const getAges = (people) => {
-  // your code here
+  const ages = people.map((person) => {
+    return getProperty('age',person);
+  });
+  return (ages);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  
```

Comment:
Have a got at using `people.find()`. It takes a callback function so it's very similar to `map()` and `forEach()`